src/main/webapp/create-live-stream.html

```diff
@@ -5,24 +5,42 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" href="css/style.css"/>
     <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
     <title>shef</title>
   </head>
  
   <body>
-    <form method="POST" action="/new-live-stream" class="form-group">
-      <h4>1.</h4>
-      <div class="form-group">
-        <label for="recipe-selection">Recipe associated with live stream:</label>
-        <select multiple class="form-control" id="recipe-selection-input" name="recipe-selection">
-          <option>Recipe 1</option>
-          <option>Recipe 2</option>
-          <option>Recipe 3</option>
-        </select>
+    <!--Nav bar-->
+    <div id="nav-placeholder">
+    </div>
+    <script>
+      $(function(){
+        $("#nav-placeholder").load("nav.html");
+      });
+    </script>
+
+    <div class="jumbotron">
+      <h1 class="display-4">Create a Livestream</h1>
+      <p class="lead">Cooking video, but make it real time.</p>
+      <hr class="my-4">
+      <button id="auth-button" class="btn btn-primary btn-lg" onclick="authenticate().then(loadClient);">Authorize</button>
+      <div id="livestream-creation">
+        <form class="form-group">
+          <h4>1. Choose a recipe to attach your live stream to.</h4>
+            <select multiple class="form-control" id="recipe-selection" name="recipe-selection">
+              <option>Recipe 1</option>
```

Comment:
Eventually, this will be populated with every recipe the user has made, correct? Or maybe the most recent ones?

Reply:
yep!

Reply:
just done in newest PR